tutorials/monitoring/tutorial.md

```diff
@@ -0,0 +1,141 @@
+---
+title: Monitoring NServiceBus solutions
+reviewed: 2017-10-02
+summary: Getting started with the monitoring capabilities built in to the Particular Service.
+---
+
+This tutorial will guide you through getting started with the monitoring capabilities built in to the Particular Service Platform.
+
+You can follow this tutorial to configure any NServiceBus based solution for monitoring with the Particular Service Platform. If you do not have a solution to monitor just yet, you can get started by downloading our sample solution.
+
+downloadbutton
+
+
+## Component overview
+
+The following diagram shows all the components involved in monitoring an NServiceBus solution with the Particular Service Platform.
+
+```mermaid
+graph TD
+
+subgraph Endpoint1
+  metrics1(NServiceBus.Metrics) --> plugin1
+  plugin1(NServiceBus.Metrics.ServiceControl)
+end
+
+subgraph Endpoint2
+  metrics2(NServiceBus.Metrics) --> plugin2
+  plugin2(NServiceBus.Metrics.ServiceControl) 
+end
+
+plugin1 --> q(Monitoring Queue)
+plugin2 --> q(Monitoring Queue)
+
+q --> scmonitoring[Monitoring Instance]
+
+scmonitoring --> pulse[ServicePulse]
+```
+
+![the above renders like this](component-overview.png)
+
+Each endpoint in the solution is configured to collect metrics about it's own operation using the NServiceBus.Metrics NuGet package. This package hooks into the NServiceBus pipeline to calculate and expose key metric data. It also adds headers to outgoing messages to track queue length.
+
+The NServiceBus.Metrics.ServiceControl NuGet package collects the generated metric data in memory and periodically sends it to a centralized monitoring queue. It does this using the same transport that the endpoints use to exchange business messages.
+
+A ServiceControl Monitoring Instance reads the messages from the monitoring queue and aggregates monitoring data for all of your endpoints. Each environment should have it's own Monitoring Instance. Monitoring Instances are created and managed with the ServiceControl Management Utility. 
+
+ServicePulse connects to a ServiceControl Monitoring Instance and visualizes the monitoring data for all of the endpoints in your environment.
+
+
+## Setting up
+
+In order to configure an environment for monitoring with the Particular Service Platform you will need to install and configure the components in the order listed.
+
+
+### Create Monitoring Instance
+
+ServiceControl is a suite of back-end tools that collect useful information about your running system.
+
+Download and run the latest ServiceControl installer. This installer contains the ServiceControl Management Utility which can be used to create and manage Monitoring Instances. 
+
+SCREENSHOT - ServiceControl Installer
+
+TODO: For the pre-release version it is currently required to add a config file with a feature toggle to the SCMU. We should either document this step or remove it.
```

Comment:
I don't think we have anything specific in the interation backlog to change that. As a result I would vote to document this step.

Reply:
Given that there is no other config required for SCMU, should we just allow users to download the file? 

I think it would be rather trivial to remove, especially if we add https://github.com/Particular/LaunchWave.DevOpsAdvancedMonitoring/issues/250#issuecomment-333761759 which gives us the opportunity to add an inline "PRE RELEASE" tag to it

Reply:
So the flow would be:
 * Download SC installer
 * Download `App.config` and put it in SCMU folder
 * After that the `Add instance` screen would show SC.Monitoring with `PRE Release` tag

If I'm getting this right, I'm 👍